pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Used for reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM input file to BAM stream and save to file:
+
+`samtools view -S -b {{input.sam}} > {{output.bam}}`
+
+- Take input from stdin (-) and print the SAM header and any reads overlapping a specific region to stdout:
+
+`{{other_command}} | samtools view -h - chromosome:start-end`
+
+- Sort file and save to BAM (the output format is automatically determined from the output file's extension):
+
+`samtools sort {{input}} -o {{output.bam}}`
+
+- Index a sorted BAM file (creates {{sorted_input.bai}}):
```

Comment:
The input file can be `.bam` or `.cram` format, but yes it probably should be `sorted_input.bam` as I specify _BAM_ in the description (plus this is by far the most common). 👍